packages/react-saasify/src/themes/waves/index.js

```diff
@@ -0,0 +1,39 @@
+import styles from './waves.module.css'
+import cv from 'css-var'
+import waveSvg from './assets/wave.svg'
+import waveShadowedSvg from './assets/wave-shadowed.svg'
+
+export const waves = ({
+  backgroundImage,
+  buttonStyle = 'normal',
+  color = '#5061CB',
+  ...opts
+} = {}) => {
+  // Define root CSS vars
+  cv({
```

Comment:
i had no idea you could do this. extremely powerful & definitely want to use this going forwards for future templates.